README.md

```diff
@@ -27,6 +31,7 @@ $.ajax({
 ```
 ### Question 5
 In an AJAX POST request, what does the argument of the `.done()` callback - in the example below, that would be `data` - represent? (Hint: it's not the same as in Question 4.)
+data represents the information entered of one user (Anna)
```

Comment:
You're right, and both times, `data` is the response coming back from the server.